src/components/app.jsx

```diff
@@ -1,5 +1,10 @@
-import React from "react";
+import React from 'react';
+import Search from './search';
 
-const App = () => <div>Hello World</div>;
+class App extends React.Component {
+  render() {
+    return <Search>Search Bar</Search>;
```

Comment:
This would be better like `const App = () => <Search />`

You're not using state so a stateless component would be better. And the `Search Bar` text isn't doing anything in the `Search` component